crates/re_types_builder/src/bin/build_re_types.rs

```diff
@@ -61,28 +69,41 @@ fn main() {
         .unwrap();
 
     let re_types_source_hash_path = workspace_dir.join(RE_TYPES_SOURCE_HASH_PATH);
-    let re_types_builder_source_hash_path = workspace_dir.join(RE_TYPES_BUILDER_SOURCE_HASH_PATH);
```

Comment:
removing the re_types_builder hash seems wrong: Whenever the builder code changes whatever hash we had for inputs from extensions and fbs files is not enough to look at.
This needs to be part of the single hash now.